api/capi/src/nnstreamer-capi-single.c

```diff
@@ -73,6 +74,16 @@ ml_single_open (ml_single_h * single, const char *model,
   /* init null */
   *single = NULL;
 
+  if (FALSE == gst_init_check (NULL, NULL, &err)) {
+    if (err) {
+      ml_loge ("GStreamer has the following error: %s", err->message);
+      g_error_free (err);
+    } else {
+      ml_loge ("Cannot initialize GStreamer. Unknown reason.");
+      return ML_ERROR_STREAMS_PIPE;
```

Comment:
move this to line 85.

if (err) cannot return error-code.

Reply:
Right! thanks